### PR TITLE
R# cleanup on design projects

### PR DESCRIPTION
**Pull request by natemcmaster**
Ran R# cleanup on all `*.Design.*` projects

cc @lajones

### CODE REVIEW

File: test/EntityFramework.MicrosoftSqlServer.Design.FunctionalTests/ReverseEngineering/SqlServerE2ETests.cs [hunk 1]
```diff
@@ -57,51 +58,52 @@ public SqlServerE2ETests(SqlServerE2EFixture fixture, ITestOutputHelper output)
...
+            }
         };
 
         // weird extenstion method call because the compiler can't disambiguate without adding a project reference
         // ApplyConfiguration swaps out the Server if this tests are configured to run against something different that localdb.
-        private string _connectionString = SqlConnectionStringBuilderExtensions.ApplyConfiguration(
-            new SqlConnectionStringBuilder(@"Data Source=(LocalDB)\MSSQLLocalDB;Initial Catalog=SqlServerReverseEngineerTestE2E;Integrated Security=True;MultipleActiveResultSets=True;Connect Timeout=30"))
-                .ConnectionString;
+        // ReSharper disable once InvokeAsExtensionMethod
+        private readonly string _connectionString = SqlConnectionStringBuilderExtensions.ApplyConfiguration(new SqlConnectionStringBuilder(@"Data Source=(LocalDB)\MSSQLLocalDB;Initial Catalog=SqlServerReverseEngineerTestE2E;Integrated Security=True;MultipleActiveResultSets=True;Connect Timeout=30"))
```
Comment (lajones) [inline test/EntityFramework.MicrosoftSqlServer.Design.FunctionalTests/ReverseEngineering/SqlServerE2ETests.cs]: Really? That has become a very long line. Isn't it possible to break it in some way that R# finds compatible? Maybe put everything after the = on a newline and see what happens?

File: test/EntityFramework.Sqlite.Design.FunctionalTests/SqliteScaffoldingModelFactoryTest.cs [hunk 2]
```diff
@@ -201,7 +201,7 @@ FOREIGN KEY (ParentId) REFERENCES Parent (Id)
             GetModel(sql);
 
             Assert.Contains("Warning: " +
-                RelationalDesignStrings.ForeignKeyScaffoldErrorPropertyNotFound("Children(ParentId)"),
+                            RelationalDesignStrings.ForeignKeyScaffoldErrorPropertyNotFound("Children(ParentId)"),
```
Comment (lajones) [inline test/EntityFramework.Sqlite.Design.FunctionalTests/SqliteScaffoldingModelFactoryTest.cs]: That's kinda weird compared to the next line. Maybe put "Warning"... on a newline?

**Comment by lajones** [comment]
Couple of minor comments. But basically :shipit:

**Comment by natemcmaster** [comment]
Ok. Will update before merging.

**Pull request merged**